app/views/vacancies/_job_details_section.html.haml

```diff
@@ -9,17 +9,12 @@
             = t('jobs.job_roles')
           %td.govuk-table__cell
             = @vacancy.show_job_roles
-      - if @vacancy.any_subjects?
+      - if @vacancy.subjects&.any?
         %tr.govuk-table__row
           %th.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
-            = t('jobs.subject', count: @vacancy.subject_count)
+            = t('jobs.subject', count: @vacancy.subjects.count)
```

Comment:
maybe you want to use again the ruby safe operator like you did on line 12 `@vacancy.subjects&.count`?